handlers/update_resource.go

```diff
@@ -2,29 +2,32 @@ package handlers
 
 import (
 	"encoding/json"
+	"errors"
 
 	"github.com/go-openapi/runtime/middleware"
 	"github.com/sul-dlss-labs/taco/datautils"
 	"github.com/sul-dlss-labs/taco/db"
 	"github.com/sul-dlss-labs/taco/generated/models"
 	"github.com/sul-dlss-labs/taco/generated/restapi/operations"
+	"github.com/sul-dlss-labs/taco/identifier"
 	"github.com/sul-dlss-labs/taco/validators"
 )
 
 // NewUpdateResource -- Accepts requests to update a resource.
-func NewUpdateResource(database db.Database, validator validators.ResourceValidator) operations.UpdateResourceHandler {
-	return &updateResourceEntry{database: database, validator: validator}
+func NewUpdateResource(database db.Database, validator validators.ResourceValidator, identifierService identifier.Service) operations.UpdateResourceHandler {
+	return &updateResourceEntry{database: database, validator: validator, identifierService: identifierService}
```

Comment:
It doesn't appear that `identifierService` is ever used in this file

Reply:
Ah, shoot, that's right. I just need a new UUID... will remove.